wildwood/tree.py

```diff
@@ -214,6 +215,72 @@ def get_nodes(self):
     def path_leaf(self, X):
         return _path_leaf(self._tree, X)
 
+    @property
+    def dirichlet(self):
+        return self._dirichlet
+    @dirichlet.setter
+    def dirichlet(self, val):
+        # we skip the checks and assume they are done by Forest
+        # higher in the hierarchy
+        if val != self.dirichlet:
```

Comment:
C'est déjà testé dans la forêt non ? Du coup on peut aussi skipper ce test là

Reply:
Oui c'est vrai